packages/remix-gcloud-functions-gen2/server.ts

```diff
@@ -0,0 +1,164 @@
+import type { NextFunction } from "express";
+import type { AppLoadContext, ServerBuild } from "@remix-run/server-runtime";
+import type {
+  Request as GcfRequest,
+  Response as GcfResponse,
+} from "@google-cloud/functions-framework";
+import type { 
+  RequestInit as NodeRequestInit,
+  Response as NodeResponse
+ } from "@remix-run/node";
+import {
+  createRequestHandler as createRemixRequestHandler,
+  Headers as NodeHeaders,
+  Request as NodeRequest,
+  writeReadableStreamToWritable
+} from "@remix-run/node";
+import {
+  // This has been added as a global in node 15+
+  AbortController,
+} from "@remix-run/node";
+import { Readable } from 'stream'
+
+/**
+ * A function that returns the value to use as `context` in route `loader` and
+ * `action` functions.
+ *
+ * You can think of this as an escape hatch that allows you to pass
+ * environment/platform-specific values through to your loader/action.
+ */
+export interface GetLoadContextFunction {
+  (req: GcfRequest, res: GcfResponse): AppLoadContext;
+}
+
+//export type RequestHandler = ReturnType<typeof createRequestHandler>;
```

Comment:
```suggestion
```

Reply:
It should be exported instead

```suggestion
export type RequestHandler = ReturnType<typeof createRequestHandler>;
```

Reply:
it's exported on the next line already (35)